src/static/js/header.js

```diff
@@ -353,6 +353,19 @@ header = {
      */
     quicklinks_html:  function() {
         let s = "";
+
+        let favouritereports = config.str(asm.user + "_FavouriteReportsID").split(",");
+        if (favouritereports) {
+            console.log(favouritereports);
+            s += "<a><span class='asm-icon asm-icon-report'></span> <select>";
+            for ( let a = 0; a < favouritereports.length; a++ ) {
+                //get_field(rows, id, field)
+                s += "<option value='" + favouritereports[a].split("=")[0] + "'>" + favouritereports[a].split("=")[1] + "</option>";
+                //s += "href='report_criteria?id=" + favouritereports[0] + "&target=report'>";
+            }
+            s += "</select></a>";
+        }
```

Comment:
Rather than using a select, use an asmmenu widget from common_widgets.js - the HTML will be something like:

```
<div id="asm-menu-quickreports" class="asm-menu-icon">Quick Reports</div>
<div id="asm-menu-quickreports-body" class="asm-menu-body">

<ul class="asm-menu-list">
<li class="asm-menu-item"><a href="report?id=1">Report 1</a></li>
</ul>

</div>
```

This is what header.js uses to make the menus at the top.